modules/ding_base/ding_base.module

```diff
@@ -750,3 +757,139 @@ function ding_base_validate_email($email) {
   $domain = strtoupper(substr($email, strrpos($email, '.') + 1));
   return empty($tld_list) || in_array($domain, $tld_list);
 }
+
+/**
+ * Implements hook_views_pre_view().
+ */
+function ding_base_views_pre_view(&$view, &$display_id, &$args) {
```

Comment:
Are you using this hook to add filter to the admin view and are you sure this is the correct hook to use?

In the documentation it's mentioned that this hook is only run for previews: https://api.drupal.org/api/views/views.api.php/function/hook_views_pre_view/7.x-3.x

FYI: The BPI module contains an example of adding filters to the admin node view via `hook_views_default_views_alter()`

Reply:
If it's not the case a @holt83 writes please add an comment to the code about why this is used and not the other hook.

Reply:
+ Good point. Thanks, fixed this.